lib/std/Build/Step/Run.zig

```diff
@@ -1004,8 +1004,8 @@ fn runCommand(
                     if (b.enable_wasmtime) {
                         try interp_argv.append(bin_name);
                         try interp_argv.append("--dir=.");
-                        try interp_argv.append(argv[0]);
                         try interp_argv.append("--");
```

Comment:
it looks like the `--` arg should be removed

Reply:
I kept it here for compatibility. Added a comment explaining the rationale.